Tests/Fixtures/Generated/Strings/flat-swift3/plurals-advanced.swift

```diff
@@ -0,0 +1,48 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+import Foundation
+
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable file_length
+
+// MARK: - Strings
+
+// swiftlint:disable function_parameter_count identifier_name line_length type_body_length
+internal enum L10n {
+  /// Plural case 'other': %@ has %d ratings
+  internal static func multiplePlaceholdersStringInt(_ p1: Int) -> String {
+    return L10n.tr("LocPluralAdvanced", "multiple.placeholders.string-int", p1)
```

Comment:
If this one is supposed to be `"%@ has %d ratings"` and thus have 2 placeholders, `String` and `Int`, then how come it only takes a single parameter of type `Int` instead of `(_ p1: String, _ p2: Int)`? Seems like a bug to me?